functions/system.bash

```diff
@@ -213,6 +213,37 @@ misc_system_settings() {
   echo "OK"
 }
 
+## change system swap size dependent on free space on /
+## swap on SD (/var/swap per default) only used after ZRAM swap full if that exists
+##
+##    change_swapsize(int size in MB)
+##
+change_swapsize() {
+  local totalMemory
+
+
+  if ! is_raspbian; then return 0; fi
+
+  totalMemory="$(grep MemTotal /proc/meminfo | awk '{print $2}')"
+  if [ -z "$totalMemory" ]; then return 1; fi
+
+  swap=$((2*totalMemory))
+  minfree=$((2*swap))
+  free=$(df -hk / | awk '/dev/ { print $3 }')
```

Comment:
$3 is used,  free $4.

Reply:
Ouch, yep. Sun is shiny, and cold beer too much of a distraction ;-)
That's what code reviews are good for - thanks.